msi/release.sh

```diff
@@ -140,6 +140,7 @@ SCRIPT_WINPATH="$(cd "$SCRIPT_PATH" && pwd -W | tr / \\\\)"
 BUILD_EXTRA_WINPATH="$(cd "$SCRIPT_PATH"/.. && pwd -W | tr / \\\\)"
 (cat <<EOF
 <?xml version="1.0" encoding="utf-8"?>
+<?define SixtyFourBit$(test -z "${BITNESS#32}" || echo =${BITNESS#32}) ?>
```

Comment:
Actually, I don't need this anymore. Since last time around, you had me create a property in the main `GitProduct.wxs` file (see the hunk above in this same commit), we use the defined pre-processor variable from `GitProduct.wds`. So this just goes away. So that was a _very_ good suggestion to create that property. :smile: